### DISCUSSION TITLE
Portal Locations for Expedition 11

### DISCUSSION THREAD

**Discussion by bedev0001** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#39] [General]
Hello I justed wanted to give some people updated portal coords, because many planets have changed fauna and flora due to the recent updates. 

Phase 2:
Corrosive Blood
![Corrosive Blood](https://github.com/user-attachments/assets/f94b91fd-3357-48e0-b3e0-10bc570aa55d)

Hunter/Killer
![HunterKiller](https://github.com/user-attachments/assets/c83fd042-af74-444a-bcef-807a3b2eb4f3)

Phase 3:
A shell to make whole
![A shell to make whole](https://github.com/user-attachments/assets/7d966193-f423-4dc3-8114-e346e3a1ad68)

Phase 5:
Hooves of Thunder
![Hooves of Thunder](https://github.com/user-attachments/assets/47a208d0-3bbb-440d-9340-954e9c81b37d)

These coords can be used by the portal that you are intended to use in phase 4 / anomalous travel. This can maybe added to the expedition 11 file
Kind regards, Hamon

**Comment by cwmonkey** [comment]
Thanks for taking the time to post this!

**Reply by bedev0001** [reply]
No problem, I just made some screenshots while I grinded these redicilous milestones off. Took me some hours

**Comment by cwmonkey** [comment]
Oof, looks like Worlds II got rid of the aggressive creature (at least). The other ones I just found by scanning the planets I went to during the course of the expedition. I don't have the patience to look for the last one so I'm just going to end this stupid expedition.

**Reply by bedev0001** [reply]
Oh no, what a shame. Very sorry for you, I took hours to find all of them, so I fully understand you. Maybe in future, it can be replayed as Redux again.
